src/patternfly/components/DataToolbar/data-toolbar-item-search-filter.hbs

```diff
@@ -0,0 +1,11 @@
+{{#> data-toolbar-item data-toolbar-item--modifier=(concat 'pf-m-search-filter ' data-toolbar-item-search-filter--modifier)}}
```

Comment:
🔥

Reply:
Looks like you're allowing for `data-toolbar-item-search-filter--modifier` in this item modifier partial, but not the others (bulk select, chip group, pagination, separator). You also support `data-toolbar-group-toggle-group--modifier` in the toggle group modifier partial. Do you think we should add it to the others for consistency?